server/game/core/Game.js

```diff
@@ -443,40 +443,47 @@ class Game extends EventEmitter {
     // }
 
     // /**
-    //  * Check to see if either player has won/lost the game due to honor (NB: this
-    //  * function doesn't check to see if a conquest victory has been achieved)
+    //  * Check to see if a base(or both bases) has been destroyed
     //  */
-    // checkWinCondition() {
-    //     let honorRequiredToWin = this.gameMode === GameMode.Skirmish ? 12 : 25;
-    //     for (const player of this.getPlayersInFirstPlayerOrder()) {
-    //         if (player.honor >= honorRequiredToWin) {
-    //             this.recordWinner(player, 'honor');
-    //         } else if (player.opponent && player.opponent.honor <= 0) {
-    //             this.recordWinner(player, 'dishonor');
-    //         }
-    //     }
-    // }
+    checkWinCondition() {
+        let losingPlayers = [];
+        for (const player of this.getPlayers()) {
```

Comment:
Bit simpler as `losingPlayers = this.getPlayers().filter((player) => player.base.damage >= player.base.getHp())`